tasks/Utility/Utils.wdl

```diff
@@ -2350,8 +2350,8 @@ task FailWithWarning {
     }
     runtime {
         cpu:                    default_attr.cpu_cores
-        memory:                 default_attr.mem_gb + " GiB"
-        disks: "local-disk " +  default_attr.disk_gb + " HDD"
+        memory:                 select_first([default_attr.mem_gb]) + " GiB"
+        disks: "local-disk " +  select_first([default_attr.disk_gb]) + " HDD"
```

Comment:
This is due to miniwdl not liking concatenating String with any optional variable. So we make the optional variable to none-optional using select_first([]). Its a little hack, happy be hear alternatives.